src/components/Dialogs/DialogManager.js

```diff
@@ -32,22 +23,24 @@ const DialogManager = () => {
 
     const onClose = () => dispatch(acSetUiOpenDimensionModal(null))
 
-    if (isDynamicDimension(dimension?.dimensionType)) {
-        return <DynamicDimension dimension={dimension} onClose={onClose} />
-    }
-    if (dimension?.dimensionType === DIMENSION_TYPE_PERIOD) {
-        return <PeriodDimension dimension={dimension} onClose={onClose} />
+    if (!dimension?.id) {
+        return null
     }
-    switch (dimension?.id) {
-        case DIMENSION_TYPE_PROGRAM_STATUS:
-        case DIMENSION_TYPE_EVENT_STATUS:
```

Comment:
the handling of `case DIMENSION_TYPE_PROGRAM_STATUS:` and `case DIMENSION_TYPE_EVENT_STATUS:` is missing